RocksDB 6.14 compilation error

the NewLRUCache() func expecting uint 

../../go/pkg/mod/github.com/blevesearch/blevex@v1.0.0/rocksdb/config.go:288:33: cannot use int(lcc) (type int) as type uint64 in argument to gorocksdb.NewLRUCache

--- a/rocksdb/config.go
+++ b/rocksdb/config.go
@@ -285,7 +285,7 @@ func applyConfig(o *gorocksdb.Options, config map[string]interface{}) (
 
 	lcc, ok := config["lru_cache_capacity"].(float64)
 	if ok {
-		c := gorocksdb.NewLRUCache(int(lcc))
+		c := gorocksdb.NewLRUCache(uint64(lcc))
 		bbto.SetBlockCache(c)
 	}
 

